frontend/src/metabase/components/DateMonthYearWidget/DateMonthYearWidget.styled.tsx

```diff
@@ -29,10 +30,15 @@ export const MonthRoot = styled.div<MonthRootProps>`
   padding: 0.5rem 1rem;
   border-radius: 99px;
   color: ${props => props.isSelected && color("text-white")};
-  background-color: ${props => props.isSelected && color("brand")};
+  background-color: ${props =>
```

Comment:
Style the hover state

#### Before
![Screenshot 2024-08-01 at 12 29 32 AM](https://github.com/user-attachments/assets/9be676b6-d1f5-4360-be42-789e6854a969)

#### After
![image](https://github.com/user-attachments/assets/c2cd4e88-eeaf-4e20-bac3-b75829b27d91)